doc/release-notes.md

```diff
@@ -4,83 +4,4 @@ release-notes at release time)
 Notable changes
 ===============
 
-RPC Changes
```

Comment:
These changes should not be included on the `version-5.5.0` branch. Please revert them.

Reply:
Actually, this just needs to be rebased, I think.